.github/workflows/release-please.yml

```diff
@@ -0,0 +1,16 @@
+name: Publish release
```

Comment:
Is `.github/workflows/release-please.yml` still needed? The build process defined in the netlify.com for this site should trigger the auto-deploy of the website when there's a push to the main branch.

Reply:
Given that the Netlify settings for this site specify the image and command to use for building the site, it seems you're correct that we don't need to keep this action